google/resource_compute_forwarding_rule.go

```diff
@@ -732,11 +732,30 @@ func expandComputeForwardingRuleSubnetwork(v interface{}, d *schema.ResourceData
 }
 
 func expandComputeForwardingRuleTarget(v interface{}, d *schema.ResourceData, config *Config) (interface{}, error) {
-	f, err := parseRegionalFieldValue("targetPools", v.(string), "project", "region", "zone", d, config, true)
+	if v == nil || v.(string) == "" {
+		return "", nil
+	} else if strings.HasPrefix(v.(string), "https://") {
+		return v, nil
+	} else if strings.HasPrefix(v.(string), "projects/") {
+		return "https://www.googleapis.com/compute/v1/" + v.(string), nil
+	} else if strings.HasPrefix(v.(string), "regions/") {
+		url, err := replaceVars(d, config, "https://www.googleapis.com/compute/v1/projects/{{project}}/")
+		if err != nil {
+			return nil, err
+		}
+		return url + v.(string), nil
+	} else if strings.HasPrefix(v.(string), "computeForwardingRule") {
+		url, err := replaceVars(d, config, "https://www.googleapis.com/compute/v1/projects/{{project}}/regions/{{region}}/")
+		if err != nil {
+			return nil, err
+		}
+		return url + v.(string), nil
+	}
+	url, err := replaceVars(d, config, "https://www.googleapis.com/compute/v1/projects/{{project}}/regions/{{region}}/computeForwardingRule/")
```

Comment:
Wait hold up- we want `computeForwardingRule` in this URL? Shouldn't the target be something that isn't a forwarding rule?

Reply:
Yeah, I don't like that rule either.  Got rid of it.  Working its way through now.

Reply:
Haha, yikes - it was redirecting the backend service to a forwarding rule also.  Yep, good that that's fixed.